tce/src/bintools/Compiler/tcecc.in

```diff
@@ -317,7 +317,7 @@ def processInputFiles(inFiles, tmpDir, options, tceopsDir=None):
 
                         command = ("clang %s %s " + \
                                        "-emit-llvm -O3 "\
-                                       "-fno-inline -c -I" + tmpDir +\
+                                       "-c -I" + tmpDir +\
```

Comment:
We perform aggressive inlining later in the optimization stage after internalizing. Did this change really affect something? If so, which case?